spec/support/meta_tags.rb

```diff
@@ -18,3 +18,28 @@
     expect(page).to have_css("meta[property='og:title'][content='Zhe title']", visible: false)
   end
 end
+
+RSpec.shared_examples "it has meta tags for images" do |schema, base_path|
+  before do
+    example_doc = GovukSchemas::RandomExample.for_schema(frontend_schema: schema) do |random|
+      random["details"] = random["details"].merge(
```

Comment:
Minor: I think you can do `random["details"].merge!(...)` to change the original object rather than an assign.